### DISCUSSION TITLE
Add drawn annotations for notes/ mark-ups

### DISCUSSION THREAD

**Discussion by Intuos5** [rbreu/beeref#92] [Ideas & General Discussion]
It would be nice to be able to draw simple annotations to the canvas, e.g. to draw relationship lines between images, arrows or to encircle features you like from a reference image. It could be a simple vector based object like Blender's Grease Pencil.

**Comment by Cinderflame-Linear** [comment]
I took a stab at it. My PR is up... just waiting on a review.

https://github.com/rbreu/beeref/pull/150
